pyproject.toml

```diff
@@ -60,7 +60,10 @@ python = "^3.6.2"
 questionary = "^1.4.0"
 decli = "^0.5.2"
 colorama = "^0.4.1"
-termcolor = "^1.1"
+termcolor = [
+    { "version" = "^1.1", python = "< 3.7" },
+    { "version" = ">= 1.1, < 2.1", python = ">= 3.7" },
```

Comment:
Wondering why are we pinning version to < 2.1.

Reply:
My bad. To follow the original caret style's intent, this should be:

```
termcolor = [
    { "version" = "^1.1", python = "< 3.7" },
    { "version" = ">= 1.1, < 3", python = ">= 3.7" },
```

If you agree, I will update the PR accordingly. Do you prefer an additional commit or should I modify and force-push?

Reply:
modify and force push